notebooks/UserSearchPy3.ipynb

```diff
@@ -1328,6 +1331,41 @@
     "lines_to_next_cell": 1
    },
    "outputs": [],
+   "source": [
+    "def prune_hits_to_ignore(full_list, id_to_find):\n",
```

Comment:
This code is what reduces false positives. Portions were present in the following cell, and refactored into this function.